src/wp-includes/class-wp-theme-json-resolver.php

```diff
@@ -425,7 +430,6 @@ public static function get_user_data_from_wp_global_styles( $theme, $create_post
 
```

Comment:
Tests are failing due to the if statement above this line (but outside the diff), `$theme` isn't defined.